File: testdata/jennies/builders/constructor_argument/JavaBuilders/src/main/java/com/grafana/foundation/sandbox/SomeStruct.java
package com.grafana.foundation.sandbox;


public class SomeStruct {
    public String title;
    
    public static class Builder {
        private SomeStruct internal;
        
        public Builder(String title) {
            this.internal = new SomeStruct();
    this.internal.title = title;
        }
    public Builder setTitle(String title) {
    this.internal.title = title;
        return this;
    }
    public SomeStruct build() {
            return this.internal;
        }
    }
}

File: testdata/jennies/rawtypes/disjunctions/CSharpRawTypes/src/Grafana/Foundation/Disjunctions/SomeStruct.cs
namespace Grafana.Foundation.Disjunctions;


public class SomeStruct
{
    public string Type;
    public object FieldAny;

    public SomeStruct()
    {
        this.Type = "";
        this.FieldAny = new object();
    }

    public SomeStruct(string type, object fieldAny)
    {
        this.Type = type;
        this.FieldAny = fieldAny;
    }
}
